sunpy/cm/cm.py

```diff
@@ -133,12 +142,13 @@ def get_cmap(name):
         raise ValueError("Colormap {name!s} is not recognized".format(name=name))
 
 
-def show_colormaps():
+def show_colormaps(mission=None):
     """Displays a plot of the custom color maps supported in SunPy.
 
     Parameters
     ----------
-    None : none
+    mission : string
+        Show only the color tables from a mission.
 
```

Comment:
This functionality is more general than just 'mission', right?  I can put in any search string.  

sunpy.cm.show_colormaps(mission='sji')

works, as does
sunpy.cm.show_colormaps('sji')

Is this intended?  Also

sunpy.cm.show_colormaps(mission="LASCO" )

does not work, so maybe some case-casting would be useful.